src/pages/registration/resource_type_tab/components/RevisionField.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTranslation } from 'react-i18next';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { ResourceFieldNames } from '../../../../types/publicationFieldNames';
+import { Checkbox, FormControlLabel } from '@mui/material';
+import { BookRegistration, Revision } from '../../../../types/publication_types/bookRegistration.types';
+import { dataTestId } from '../../../../utils/dataTestIds';
+
+export const RevisionField = () => {
+  const { setFieldValue } = useFormikContext<BookRegistration>();
+  const { t } = useTranslation();
+
+  return (
+    <Field name={ResourceFieldNames.Revision}>
+      {({ field }: FieldProps) => (
```

Comment:
god praksis å angi typen på Field også. Da ser du at du får riktig type på `field.value` på linje 19, i stedet for `any`. Om typen endrer seg i ettertid vil man (forhåpentligvis) se at noe brekker her også
```suggestion
      {({ field }: FieldProps<Revision | null | undefined>) => (
```